AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/vh-participant.ts

```diff
@@ -0,0 +1,126 @@
+import { InterpreterSelectedDto } from 'src/app/booking/interpreter-form/interpreter-selected.model';
+import { ScreeningDto } from 'src/app/booking/screening/screening.model';
+import { LinkedParticipantModel } from './linked-participant.model';
+import { HearingRoleCodes, HearingRoles } from './hearing-roles.model';
+import { v4 as uuid } from 'uuid';
+import { cloneWithGetters } from '../helpers/clone-with-getters';
+
+export class VHParticipant {
+    id?: string;
+    externalReferenceId?: string;
+    title?: string;
+    firstName?: string;
+    lastName?: string;
+    middleNames?: string;
+    displayName?: string;
+    username?: string;
+    email?: string;
+    hearingRoleName?: string;
+    hearingRoleCode?: string;
+    phone?: string;
+    representee?: string;
+    company?: string;
+    isExistPerson?: boolean;
+    interpreterFor?: string;
+    linkedParticipants?: LinkedParticipantModel[];
+    interpreteeName?: string;
+    isInterpretee?: boolean;
+    userRoleName?: string;
+    isCourtroomAccount?: boolean;
+    addedDuringHearing?: boolean;
+    contactEmail?: string;
+    isJudiciaryMember?: boolean;
+    interpretation_language: InterpreterSelectedDto;
+    screening?: ScreeningDto;
+
+    constructor(init?: Partial<VHParticipant>) {
+        Object.assign(this, init);
+        if (!this.externalReferenceId) {
+            this.externalReferenceId = uuid();
+        }
+    }
+
+    get fullName(): string {
```

Comment:
I wouldn't typically suggest xml docs on fields but it might be worth clearing up that full name include Title

Reply:
added